homeassistant/components/lupusec/__init__.py

```diff
@@ -85,10 +106,11 @@ def __init__(self, username, password, ip_address, name):
 class LupusecDevice(Entity):
     """Representation of a Lupusec device."""
 
-    def __init__(self, data, device):
+    def __init__(self, data, device, config_entry=None) -> None:
         """Initialize a sensor for Lupusec device."""
         self._data = data
         self._device = device
+        self._entry_id = config_entry.entry_id if config_entry else "_none"
```

Comment:
We still don't use the config entry right?

Reply:
Yes, you are right. Too early in the morning ;)